cli/tests/testdata/lib_dom_extras.ts

```diff
@@ -0,0 +1,17 @@
+const { diagnostics, files } = await Deno.emit("/main.ts", {
+  compilerOptions: {
+    target: "esnext",
+    lib: ["esnext", "dom"],
+  },
+  sources: {
+    "/main.ts": `const as = new AbortSignal();
+    console.log(as.reason);
+    
+    const up = new URLPattern("https://example.com/books/:id");
+    console.log(up);
+    `,
+  },
+});
+
+console.log(diagnostics);
+console.log(Object.keys(files).sort());
```

Comment:
Is this test file used? It would be good to add a test for this (Maybe I’m missing something?)

Reply:
sorry, forgot to add it as an integration test... 🤦  tested it manually though